examples/blog/src/pages/blog/using-mdx.mdx

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I understand we want to keep config minimal, but adding an MDX example that doesn't show _any_ MDX features feels a little wrong. If we don't want to add a component integration, that's fair! But at least importing and using `.astro` component, or accessing frontmatter via JSX expressions, would be helpful.

Reply:
Agreed. I had the same thought. I would also add another heading under Usage with a similar very short explanation re: importing and using a component.

We do have the MDX starter already (though not as visible), so this *can* be just a Markdown starter if we want to go that route. (Is there value in having one?) I do think our "add MDX" path is straightfoward and well-documented enough that I don't worry about people adding it if they want. Just depends what you want people to get by default, using `create-astro`.

Reply:
I had originally put an `.astro` component with a https://www.npmjs.com/package/canvas-confetti button, but then realized that our `Astro.glob()` on the blog page was pulling that script into the blog page itself. This is a known side-effect of `Astro.glob()`, but it felt too problematic to bring into a starter template (at least until we fix it).

I'll do it with one of the existing header links for now.